### PR TITLE
Update Polish translation

### PR DESCRIPTION
**Pull request by hevet**
### Check if this PR fulfills these requirements:
* [ ] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
* [ ] I have read the [**CONTRIBUTING**](../../master/Contributing.md) document.
* [ ] I made sure there wasn't another [Pull Request](../../../pulls) opened for the same update/change
* [ ] I have successfully tested my changes locally

#### Types of changes
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Feature change (non-breaking change which changes behaviour of an existing functionality)
- [ ] Improvement (non-breaking change which improves functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactor (non-breaking performance or readability improvements)

### CODE REVIEW

File: resources/language/resource.language.pl_pl/strings.po [hunk 1]
```diff
@@ -220,13 +220,13 @@ msgstr "{} Ustaw do odtwarzania z biblioteki"
 # Unused 30053, 30054
 msgctxt "#30055"
 msgid "{} Set at start-up"
-msgstr ""
+msgstr "Ustaw przy uruchomieniu"
```
Comment (CastagnaIT) [inline resources/language/resource.language.pl_pl/strings.po:223]: placeholder is missing

File: resources/language/resource.language.pl_pl/strings.po [hunk 2]
```diff
@@ -220,13 +220,13 @@ msgstr "{} Ustaw do odtwarzania z biblioteki"
...
 
 #. The {} will be replaced with the graphic circle to mark when the functionality is active
 # Unused 30056
 msgctxt "#30057"
 msgid "{} Remember PIN"
-msgstr ""
+msgstr "Zapamiętaj PIN"
```
Comment (CastagnaIT) [inline resources/language/resource.language.pl_pl/strings.po:229]: placeholder is missing

**Review by CastagnaIT — CHANGES_REQUESTED** [review]
the CI notify others untranslated strings

resources/language/resource.language.pl_pl/strings.po:1127: this message is untranslated
resources/language/resource.language.pl_pl/strings.po:1131: this message is untranslated
resources/language/resource.language.pl_pl/strings.po:1157: this message is untranslated
resources/language/resource.language.pl_pl/strings.po:1170: this message is untranslated
resources/language/resource.language.pl_pl/strings.po:1175: this message is untranslated
resources/language/resource.language.pl_pl/strings.po:1185: this message is untranslated
resources/language/resource.language.pl_pl/strings.po:1190: this message is untranslated

File: resources/language/resource.language.pl_pl/strings.po (hunk 2 above)
Comment (hevet) [reply resources/language/resource.language.pl_pl/strings.po]: I corrected the translation, added the missing translations, but it was not verified. I don't know where the problem is?

Comment (CastagnaIT) [reply resources/language/resource.language.pl_pl/strings.po]: its ok now,
CI pylint its not a translation problem

**Review by CastagnaIT — APPROVED** [review]

**Pull request merged**